content/blog/introducing-python-sdk-for-dscc.md

```diff
@@ -0,0 +1,130 @@
+
+**APIs and SDKs** 
+
+APIs are a list of functions libraries that are used to communicate with a web service (via secure HTTP), whereas an SDK is a development kit that facilitates the usage of these APIs. APIs enable any software developer to create business opportunities by leveraging the capabilities provided by the API to extend their application whereas the SDK facilitates the process for developers. An SDK is a collection of software tools and programs for a specific application platform that allows developers to manipulate the functions supported by the service. These can be considered as a wrapper on the top of the APIs, making the code consumable by the application.
+
+One thing to note is that, the user always gets access to the latest Data Services Cloud Console API version through the SDKs. How? The SDK is designed and deployed (using CI/CD pipelines such as Jenkins) in such a way that with every new release of the Data Services Cloud Console Open API spec, the SDKs get updated automatically. Thus, keeping it up-to date without any manual intervention. This also reduces time which is spent waiting for updates with newer features.
+
+**Introducing Python SDK for Data Services Cloud Console**
+
+Due to the wide adoption of Python, and for the Python lovers out there who did not have an option to achieve their automation goals, we have the Python SDK available now. You can access the SDK onthis [github ](https://github.com/HewlettPackard/greenlake-data-services-python)page.
+
+This SDK contains the following:
+
+* Documentation (under docs folder)
+* Code libraries (under greenlake_data_services folder)
+* Test file (under test folder)
+* README file
+* The Python libraries that are required to run this SDK (requirements.txt & test-requirements.txt)
+
+**Requirements**:
+
+Python (>=3.5) is required to run the scripts. Run the following command to install all the required packages that the SDK is dependent on:
+
+```shell
+pip install –r requirements.txt
+```
+
+Example Usage:
+
+Let us consider Audits as an example. Audit events are a collection of tasks performed by users. The below code snippet uses a GET method to fetch the details of audit events, like task ID, user email, state, etc.
+
+The sample code is provided in the [documentation](https://github.com/HewlettPackard/greenlake-data-services-python/blob/dev/docs/AuditEventApi.md#device_type2_get_events)of this resource. Take the sample code and replace the BEARER_TOKEN with the access token. Generate the access token as mentioned in this [blog](https://developer.hpe.com/blog/oauth2-for-hpe-greenlake-data-services-cloud-console/).
```

Comment:
I think you need a space after "documentation".

![image](https://user-images.githubusercontent.com/77496261/228364122-1bb55d57-77e5-4dc6-9361-c44c9befd753.png)